packages/expo-splash-screen/src/index.native.ts

```diff
@@ -4,11 +4,20 @@ import { SplashScreenNativeModule, SplashScreenOptions } from './SplashScreen.ty
 
 const SplashModule = requireOptionalNativeModule<SplashScreenNativeModule>('ExpoSplashScreen');
 
+const isExpoGo = typeof expo !== 'undefined' && globalThis.expo?.modules?.ExpoGo;
```

Comment:
personally i would like to use a helper than having the implementation fact

```ts
import { isRunningInExpoGo } from 'expo';
```

Reply:
I forgot that was there 😅 updated